lib/server.ts

```diff
@@ -71,22 +71,24 @@ app.post('/launch', (req, res) => {
     // Create temp dir
     tmp.dir({'unsafeCleanup': true}, (err, tempDir, cleanTempDir) => {
         // Generate contents
-        deploy.apply(resources({}), tempDir, req.body.name, req.body.runtime, normalizeCapabilities(req.body.capability))
+        deploy.apply(resources({}), tempDir, req.body.name, req.body.runtime, req.body.capabilities)
```

Comment:
1. you got rid of `normalizeCapabilities`
2. You forgot to do the same for the `/zip` endpoint

Reply:
@quintesse `zip` endpoint is a http GET and use query params, so I didn't want to touch it (should I?).

Reply:
Well, they should at least be the same, shouldn't they? I mean changing `capability` to `capabilities` only in one endpoint doesn't make much sense?

The other thing is that the `normalizeCapabilities()` is there to make it easier to use from the Swagger page, otherwise you are forced to type in a whole object definition while this way it will also just accept a simple string. I'd like to keep that feature (at least for now while we do a lot of manual testing).

Reply:
Oh btw, yes, I think we should probably change the `/zip` endpoint too. They are basically just copies, just with one returning the zip while the other one pushes it on to the backend's `/upload` endpoint.